templates/control/2fa/init_verify.html

```diff
@@ -0,0 +1,56 @@
+$def with (tfa_secret, tfa_recovery_codes, error)
+$:{RENDER("common/stage_title.html", ["Enable 2FA: Step 3", "Two-Factor Authentication", "/control/2fa/status"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/2fa/init_verify">
+    $if error == "2fa":
+      <div id="signin_error">
+        <strong>Whoops!</strong> The 2FA response token you provided did not validate.
+      </div>
+    $elif error == "verify":
+      <div id="signin_error">
+        <strong>Hey!</strong> Did you want to enable 2FA? You didn't check the verification checkbox!
+      </div>
+
+    <h3>Verify Receipt of Two-Factor Authentication (2FA) Recovery Codes</h3>
+    <p>
+      <strong>Print these codes, and secure them as you would your password.</strong>
+      In the event you lose access to your authenticator app, you will need these
+      codes to regain access to your Weasyl account. Recovery codes can be
+      refreshed at any time from your settings page.
+    </p>
+    <p>
+      <strong>Warning:</strong> For security reasons, Weasyl staff will be unable to assist you if
+      you lose access to your recovery codes. We advise never using more than eight codes, as you will
+      need two to disable 2FA from a logged-out state (one code to log in, and one code to disable
```

Comment:
Technically for this one I didn't want users to rely on using all codes to disable 2FA, as it isn't the "intended" way; I did this more as a mechanism to prevent user's from locking themselves out if all codes were used. That said, I'll just remove the sentence in its entirety, since while the proper path has a user disable from a logged out state with two (login, then deactivate), there is also the backup path of logging in with one code remaining force-disables (and there's no other way to use a code while logged in except to regenerate codes (problem solved), or disable 2FA (also problem solved)).